pkg/generator/error.go

```diff
@@ -27,3 +27,12 @@ func IsNotFound(err error) bool {
 
 	return microerror.Cause(err) == notFoundError
 }
+
+var emptyValueError = &microerror.Error{
+	Kind: "emptyValueError",
+}
+
+// IsEmptyValue asserts emptyValueError.
+func IsEmptyValue(err error) bool {
+	return microerror.Cause(err) == emptyValueError
+}
```

Comment:
This error is not matched so should be replaced with `executionFailedError` without a matcher.